README.md

```diff
@@ -36,13 +36,13 @@ This now provides an accessible http or [https](#https) endpoint for flag evalua
 Command:
 
 ```sh
-curl -X POST "localhost:8013/flags/myBoolFlag/resolve/boolean"
+curl -X POST "localhost:8013/schema.v1.Service/ResolveBoolean" -d '{"flagKey":"myBoolFlag","context":{}}' -H "Content-Type: application/json"
 ```
 
 Result:
 
 ```sh
-{"value":true,"reason":"STATIC","variant":"on"}
+{"value":true,"reason":"TARGETING_MATCH","variant":"on"}
```

Comment:
I'm surprised this is a targeting match. I would have expected `DEFAULT`.

@toddbaert you had some thoughts on the reason values. What do you think?

Reply:
apologies this is a typo - ive updated this to STATIC, there is another draft PR open to change these values to DEFAULT, awaiting a conclusion to the discussion in the openfeature-flagd slack channel

Reply:
Agreed, thanks.